cafebabel/templates/articles/detail.html

```diff
@@ -34,13 +34,14 @@
     <h1>{{ article.title }}</h1>
 
     {{ macros.article_meta(article) }}
-
-    <p>
-      Tags:
-      {% for tag in article.tags -%}
-        {{ tag.name }}{% if loop.last %}.{% else %}, {% endif %}
-      {%- endfor %}
-    </p>
+    {% if article.tags|length != 0 %}
+      <p class=tags-list>
+        {% for tag in article.tags -%}
+          {#<a href={{ url_for('tags.detail', slug=tag.slug) }}>{{ tag.name }}</a>#} <!-- next PR -->
+          {{ tag.name }}
```

Comment:
Please get back to the previous version: `{{ tag.name }}{% if loop.last %}.{% else %}, {% endif %}` or change tests (or explain why you did that in the commit message)

Reply:
I want to have that in the end: ![capture d ecran 2018-01-18 a 16 30 02](https://user-images.githubusercontent.com/32518627/35105809-e500f8f6-fc6c-11e7-81ac-aa3c246fcdfa.png)

Reply:
I know, that's just not the right branch/PR.